test/v3/controllers/requestParsers/SubmitForeignPropertyBsasRequestParserSpec.scala

```diff
@@ -43,16 +44,17 @@ class SubmitForeignPropertyBsasRequestParserSpec extends UnitSpec {
       |""".stripMargin
   )
 
-  val inputData: SubmitForeignPropertyRawData =
-    SubmitForeignPropertyRawData(nino, calculationId, requestBodyJson)
+  def inputDataWith(taxYear: Option[String]): SubmitForeignPropertyRawData =
```

Comment:
Nice.. I prefer this way of doing it than having a nonTysInputData and TysInputData